packages/outline-core/src/outline-element/outline-element.a11y.css

```
@@ -6,3 +6,14 @@
   height: 1px;
   word-wrap: normal;
 }
+
+.visually-hidden,
+::slotted(.visually-hidden),
+[aria-hidden='true'] {
+  position: absolute !important;
+  overflow: hidden;
+  clip: rect(1px, 1px, 1px, 1px);
+  width: 1px;
+  height: 1px;
+  word-wrap: normal;
+}
```

I cannot add this as a suggestion in Github, because the code is on top of the file of this PR.
Instead of adding the duplicate code, just add the `.visually-hidden` class above the existing `.sr-only` class.

We should avoid assuming that every `aria-hidden` is same as visually-hidden (because it's not)
the `.sr-only` definition already have that in the correct way.

The top of this file should be -
```css
.visually-hidden,
.sr-only {
```


```suggestion

.visually-hidden,
::slotted(.visually-hidden),
[aria-hidden='true'] {
  position: absolute !important;
  overflow: hidden;
  clip: rect(1px, 1px, 1px, 1px);
  width: 1px;
  height: 1px;
  word-wrap: normal;
}
```

---

OK, I think I updated the code based on your comment. This code came from a client project where they had more than just `.visually-hidden` and also added the `::sloted(.visually-hidden)` and the `[aria-hidden='true']` selectors.  I don't know if those are really needed but kept them here.